browser-test/src/applicant/navigation/northstar_application_navigation_buttons.test.ts

```diff
@@ -228,4 +228,934 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
       })
     })
   })
+
+  test.describe('navigation with five blocks', () => {
```

Comment:
**question:** In this test, there is already a test.describe with "navigation with five blocks". Is this duplicative at all? Should we remove what's above?

Reply:
Good point. I removed the duplicated beforeEach and organized the test cases